src/interfaces/configuration-state-interface.ts

```diff
@@ -0,0 +1,27 @@
+
+/** The state model for the configuration store module. */
+export interface ConfigurationStateIF {
+  configuration: {
+    ADDRESS_COMPLETE_KEY: string,
+    AUTH_API_URL: string,
+    AUTH_API_VERSION: string
+    AUTH_WEB_URL: string,
+    BUSINESS_CREATE_URL: string,
+    BUSINESS_EDIT_URL: string,
+    BUSINESS_FILING_LD_CLIENT_ID: string,
+    BUSINESSES_URL: string,
+    DASHBOARD_URL: string,
+    HOTJAR_ID: string
+    KEYCLOAK_CONFIG_PATH: string,
+    LEGAL_API_URL: string,
+    LEGAL_API_VERSION_2: string
+    PAY_API_URL: string,
+    PAY_API_VERSION: string
+    REGISTRY_HOME_URL: string,
+    SENTRY_DSN: string,
+    SENTRY_ENABLE: string,
+    SITEMINDER_LOGOUT_URL: string,
+    STATUS_API_URL: string,
+    STATUS_API_VERSION: string
+  }
+}
```

Comment:
Just a thought about this. Did you see at the last tech demo that Patrick Wei will be changing this config system over when apps are ported to Firebase/GCP in the (near?) future, similar to [BC Registry app](https://github.com/bcgov/bcregistry) I think.

Reply:
Unfortunately, no. I was invited to the tech demo, but the meeting link wasn't included.  I'll get it sorted in time for the next meeting.  I'm not sure what Patrick has planned, but it would be great to load the configuration via the .env files.  This makes it easy to override for different environments.

Reply:
No action required here then. This will work for now, and the change to env files will be future work.